src/dialects/abstract/query-generator.js

```diff
@@ -2221,7 +2221,21 @@ export class AbstractQueryGenerator extends AbstractQueryGeneratorTypeScript {
   selectFromTableFragment(options, model, attributes, tables, mainTableAs) {
     this._throwOnEmptyAttributes(attributes, { modelName: model && model.name, as: mainTableAs });
 
-    let fragment = `SELECT ${attributes.join(', ')} FROM ${tables}`;
+    let fragment = `SELECT`;
+
+    if (typeof options.maxExecutionTimeMs === 'number' && this.dialect.supports.maxExecutionTimeHint) {
+      const MINIMUM_EXECUTION_TIME_VALUE = 0;
+      const MAXIMUM_EXECUTION_TIME_VALUE = 4_294_967_295;
```

Comment:
Where is this number coming from?

Reply:
It's in [MySQL Docs](https://dev.mysql.com/doc/refman/5.7/en/server-system-variables.html#sysvar_max_execution_time).

Reply:
Could you add a link to it so it's explicit that this is about the mysql implem? (since it's in the abstract query generator)

Reply:
I move actual logic to mysql query generator in [here](https://github.com/sequelize/sequelize/pull/15341/commits/48b32fc5cb0a3acd73b22419590ab5bbca5cbd54), so I resolve this.